index.bs

```diff
@@ -32,6 +32,29 @@ spec: ABNF; urlPrefix: https://tools.ietf.org/html/rfc5234
     text: VCHAR; url: appendix-B.1
     text: WSP; url: appendix-B.1
 
+spec: CSP; urlPrefix: https://www.w3.org/TR/CSP3/
+  type: dfn
+    text: strip url for use in reports; url: strip-url-for-use-in-reports
+
+spec: DOM; urlPrefix: https://dom.spec.whatwg.org/
+  type: dfn
+    text: document url; url: concept-document-url
+
+spec: INFRA; urlPrefix: https://infra.spec.whatwg.org
+  type: dfn
+    text: list; url: list
+
+spec: HTML; urlPrefix: https://html.spec.whatwg.org/C
+  type: dfn
+    text: policy container; url: policy-container
+    text: global object; url: global-object
+    text: window; url: window
+    text: WorkerGlobalScope; url: workerglobalscope
+
+spec: FETCH; urlPrefix: https://fetch.spec.whatwg.org/
+  type: dfn
+    text: main fetch; url: main-fetch
```

Comment:
I continue to believe that these are not needed or should instead be addressed upstream.

Reply:
Removed most of these, with two remaining: "main fetch" and "strip URL for use in reports".

I'm happy to upstream fixes for the latter (to properly export it) and happy to directly integrate with main fetch assuming that's a reasonable approach. Let me know what you think.